Model/Sales/Repository/PackageRepository.php

```diff
@@ -94,39 +112,48 @@ public function productWithoutDeliveryOptions(array $products)
     }
 
     /**
+     * @param $product
+     * @param $mailBox
+     *
      * @return bool
      */
-    public function fitInMailbox(): bool
+    public function fitInMailbox($product, $mailBox): bool
     {
+        $mailboxPercentage    = $this->getMailboxPercentage();
+        $mailboxPercentage    += $mailBox * $product->getQty();
```

Comment:
Ik vind dit duidelijker:
```suggestion
        $mailboxPercentage    = $this->getMailboxPercentage() + $mailBox * $product->getQty();
```

Reply:
Ik snap getMailboxPercentage niet. Wil je een comment plaatsen (hier en/of boven de functie)?

Reply:
Done